### ISSUE TITLE
Quizzes explanation is not accessible

### ISSUE THREAD

**Issue by amirmahdifard** [UnigramDev/Unigram#2737]
when we are answering to a quis and after selecting an option, we havving a button called (Explanation  button), and when clicking on this button, we can't read the Explanation of that quis, because it isn't accessible for screen readers. version: last version of the channel.

**Comment by FrayxRulez** [comment]
Can you try to write meaningful titles?

**Comment by amirmahdifard** [comment]
@FrayxRulez whats your mean, your mean I should tell a little bit that's whats happening in title?

**Comment by FrayxRulez** [comment]
This doesn't depend on Unigram and I've created a new issue in the right place: https://github.com/microsoft/microsoft-ui-xaml/issues/7992
